python/lsst/ctrl/pool/parallel.py

```diff
@@ -262,7 +262,9 @@ def submitCommand(self, scriptName):
         return "exec %s" % scriptName
 
 
-BATCH_TYPES = {'pbs': PbsBatch,
+BATCH_TYPES = {'none' : None,
+               'None' : None,
```

Comment:
Instead of supporting both `none` and `None`, might we always convert the key to lower case?